actionpack/lib/action_dispatch/middleware/actionable_exceptions.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require "erb"
+require "action_dispatch/http/request"
+require "active_support/actionable_error"
+
+module ActionDispatch
+  # This middleware handles the dispatching and execution of actionable errors
+  # actions.
+  class ActionableExceptions
+    cattr_accessor :endpoint, default: "/rails/actions"
```

Comment:
This should probably be a `class_attribute` if anyone wants to extend this class.

Reply:
I use the `cattr_accessor` as a configuration here. Will mark the class as `# :nodoc:` as I'd like to think of it as an implementation detail.